network/datadog_checks/network/network.py

```diff
@@ -91,8 +349,13 @@ def check(self, instance):
         self._collect_rate_metrics = instance.get('collect_rate_metrics', True)
         self._collect_count_metrics = instance.get('collect_count_metrics', False)
         self._collect_ena_metrics = instance.get('collect_aws_ena_metrics', False)
-        if fcntl is None and self._collect_ena_metrics:
-            raise ConfigurationError("fcntl not importable, collect_aws_ena_metrics should be disabled")
+        self._collect_ethtool_metrics = instance.get('collect_ethtool_metrics', False)
+
+        self._collect_ethtool_stats = self._collect_ena_metrics or self._collect_ethtool_metrics
+        if fcntl is None and self._collect_ethtool_stats:
```

Comment:
This integration is listed as available for windows, fcntl does not exist in windows

Reply:
I've moved all code related to ethtool to a dedicated module.